src/rocksdb-types/src/config.rs

```diff
@@ -138,6 +142,14 @@ pub struct RocksDBTuningParameters {
     /// If not provided there will be no optimization.
     /// <https://github.com/facebook/rocksdb/blob/main/include/rocksdb/options.h#L82-L85>
     pub point_lookup_block_cache_size_mb: Option<u32>,
+
+    /// Optional write buffer manager bytes. This needs to be set to enable write buffer manager
+    /// across all rocksdb instances
+    pub write_buffer_manager_memory_bytes: Option<usize>,
+    /// Optional write buffer manager memory limit as a percentage of cluster limit
+    pub write_buffer_manager_memory_percent: Option<usize>,
```

Comment:
Nit: I find it more intuitive to express percentages as a number `[0,1]`, which avoids division on calculating the final values, potentially with integer rounding errors.

Reply:
The struct might not need an `Eq`. Let me remove it instead, and can use fraction instead of percentage then.